From 5c12bb6e0cb8f9f8f2112d2c7d3929262b4d4427 Mon Sep 17 00:00:00 2001
From: Davide Depau <davide@depau.eu>
Date: Thu, 16 Jan 2020 23:13:24 +0100
Subject: [PATCH] Workaround for non compliant BLE lights

---
 src/shared/gatt-client.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/shared/gatt-client.c b/src/shared/gatt-client.c
index 29254cb61..4903c4ec0 100644
--- a/src/shared/gatt-client.c
+++ b/src/shared/gatt-client.c
@@ -1078,6 +1078,7 @@ static void discover_secondary_cb(bool success, uint8_t att_ecode,
 
 	if (!success) {
 		switch (att_ecode) {
+		case 0x00:
 		case BT_ATT_ERROR_ATTRIBUTE_NOT_FOUND:
 		case BT_ATT_ERROR_UNSUPPORTED_GROUP_TYPE:
 			success = true;
-- 
2.25.0

